draft-ietf-rats-architecture.md

```diff
@@ -1164,6 +1164,11 @@ Using the table above, a number of hypothetical examples of how a solution might
 a solution might be built.  This list is not intended to be complete,
 but is just representative enough to highlight various timing considerations.
 
+All times are relative to the local clocks, indicated by an "a" (Attester),
+"v" (Verifier), or "r" (Relying Party) suffix.
```

Comment:
Can you fix line 1236 to also have the quotes and capitalize Verifier there?

Reply:
Done.